.github/workflows/tests.yml

```diff
@@ -16,9 +16,14 @@ jobs:
     strategy:
       fail-fast: true
       matrix:
-        php: [7.2, 7.3, 7.4, '8.0', 8.1, 8.2]
+        php: ['8.1', '8.2']
+        deps: [highest]
 
-    name: PHP ${{ matrix.php }}
+        include:
+          - php: '8.1'
+            deps: lowest
```

Comment:
This new Job tests with low dependencies. This way, issues like the one I've just fixed can be spotted more easily.

Reply:
If we add `lowest` to `matrix.deps` should that work for both versions?

Reply:
Yes. Do you want a deps=lowest job for both PHP versions? I felt like one of those is enough.

Reply:
Ok, yes, one is enough.